processor/processor.go

```diff
@@ -122,7 +122,13 @@ func ProcessConstants() {
 	startTime := makeTimestampNano()
 	for name, value := range languageDatabase {
 		for _, ext := range value.Extensions {
-			ExtensionToLanguage[ext] = name
+			_, ok := ExtensionToLanguage[ext]
+
+			if ok {
```

Comment:
This condition looks superfluous. `append`ing to a `nil` slice will create a new slice so the

    ExtensionToLanguage[ext] = append(ExtensionToLanguage[ext], name)

line should be all we need.

Reply:
Ah neat. Never thought about that but it makes perfect sense.